src/airgap/upload_bundles.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    echo "No release number supplied as command line argument."
+    echo "Usage: ./upload_bundles.sh release_number (harness-x.x.x)"
+    exit 1
+fi
+
+release_number=$1
+
+files=("platform_images.tgz" "cdng_images.tgz" "ccm_images.tgz" "srm_images.tgz" "ce_images.tgz" "ff_images.tgz" "ci_images.tgz" "sto_images.tgz")
+
+touch empty_file
+gsutil cp empty_file gs://smp-airgap-bundles/${release_number}/
```

Comment:
Instead of copying empty_file, wouldn't copying the files directly to the provided path work?

Reply:
It would work, this approach will return sooner if there is any issue, else upload of bundle can take few hours.